test/gurka/test_landmarks.cc

```diff
@@ -2,60 +2,142 @@
 #include <gtest/gtest.h>
 
 #include "gurka.h"
-#include "mjolnir/landmark_database_builder.h"
+#include "mjolnir/landmark_builder.h"
 #include "test/test.h"
+#include <boost/property_tree/ptree.hpp>
 
 using namespace valhalla;
 using namespace valhalla::baldr;
 using namespace valhalla::gurka;
 using namespace valhalla::mjolnir;
 
-const std::string db_name = "landmarks.db";
-const std::filesystem::path file_path = db_name;
+const std::string db_name_test1 = "landmarks.db";
+const std::filesystem::path file_path_test1 = db_name_test1;
+
+namespace {
+valhalla::gurka::map BuildPBF(const std::string& workdir) {
+  const std::string ascii_map = R"(
+      a-----b-----c---d
+      A   B   C   D   E
+    )";
+
+  const gurka::nodes nodes = {
+      {"A", {{"name", ""}, {"amenity", "university"}}},
+      {"B", {{"name", "hai di lao"}, {"amenity", "restaurant"}}},
+      {"C", {{"name", "ke ji lu"}, {"amenity", ""}}}, // no amenity, shouldn't be stored
+      {"D", {{"name", "wan da"}, {"amenity", "cinema"}}},
+      {"E", {{"name", "zhong lou"}, {"amenity", "monument"}}}, // not in list, shouldn't be stored
```

Comment:
(accidentally?) you gave a VERY good example of OSM stuff we'd also like to save apart from `amenity`. See [`historic` key](https://taginfo.openstreetmap.org/keys/historic#values) which also contains `monument`. If the name is descriptive enough that's the perfect guidance POI:)

But let's shelve that for a later day: https://github.com/valhalla/valhalla/issues/4210

Reply:
you remind me it's not an accident :) I was discovering osm key-values and found `monument`, thought it could be interesting to have it as landmarks but forgot to discuss with you. no problem let's discuss later.